### PR TITLE
add jsexec to contrib

### PR DESCRIPTION
**Pull request by fm-tibco**

### CODE REVIEW

File: activity/jsexec/go.mod [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module github.com/project-flogo/jsexec
```
Comment (skothari-tibco) [inline activity/jsexec/go.mod]: shouldn't it be contrib/activity/jsexec

File: activity/jsexec/examples/microgateway/api/go.mod [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module api
```
Comment (skothari-tibco) [inline activity/jsexec/examples/microgateway/api/go.mod]: This is a sample module and not intended for other use right?

Comment (fm-tibco) [reply activity/jsexec/examples/microgateway/api/go.mod]: yes

File: activity/jsexec/go.mod (hunk 1 above)
Comment (fm-tibco) [reply activity/jsexec/go.mod]: fixed

**Pull request merged**